scripts/talos/bumpSimorghPackages/updatePackageJson/index.js

```diff
@@ -0,0 +1,30 @@
+const commitRemoteChanges = require('../../../utilities/commitRemoteChanges');
+
+const mergeUpdates = (packageJson, publishedPackages) => {
+  const newPackageJson = { ...packageJson };
+  Object.keys(publishedPackages).forEach(key => {
+    if (newPackageJson.dependencies[key])
```

Comment:
it would be nice to see this stored as a well named variable like `hasNewDependency`